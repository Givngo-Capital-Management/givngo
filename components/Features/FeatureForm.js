import React from 'react';
import styles from './FeatureForm.module.scss';

export default function FeatureForm() {
  return (
    <div className={styles['form-layout']}>
      <div className={styles['form-block']}>
        <h2 className={styles['form-header']}>
          Increase your
          <br />
          charity donations
        </h2>
        <div className={styles['form-text']}>
          Leave your contact information below, and we&apos;ll be in touch soon
          to help you get started
        </div>
        <form
          id="contact-form"
          name="email-form"
          className={styles.form}
          method="post"
        >
          <label htmlFor="name-2" className={styles['field-label']}>
            Name
            <input
              type="text"
              className={[styles['name-text-field'], styles['w-input']].join(
                ' ',
              )}
              maxLength="256"
              name="name"
              id="name-2"
              placeholder="Name"
            />
          </label>

          <label htmlFor="email-2" className={styles['field-label']}>
            Email Address
            <input
              type="email"
              className={[styles['name-text-field'], styles['w-input']].join(
                ' ',
              )}
              maxLength="256"
              name="email"
              id="email-2"
              placeholder="Email Address"
            />
          </label>
          <input
            id="contact-form-btnSubmit"
            type="submit"
            value="get started"
            data-wait="Please wait..."
            className={[styles['button-form'], styles['w-button']].join(' ')}
          />
        </form>
        <div className={styles['w-form-done']}>
          <div>Thank you! We will contact you soon!</div>
        </div>
        <div className={styles['w-form-fail']}>
          <div>
            Oops! Something went wrong while submitting the contact information,
            please try again.
          </div>
        </div>
      </div>
    </div>
  );
}
