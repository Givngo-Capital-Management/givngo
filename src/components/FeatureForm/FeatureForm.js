import React from 'react'
import './FeatureForm.scss'

export default function FeatureForm() {
  return(
    <div className="form-layout">
      <div className="form-block">
        <h2 className="form-header">Increase your<br />charity donations</h2>
        <div className="form-text">Leave your contact information below, and we'll be in touch soon to help you get started</div>
        <form id="contact-form" name="email-form" className="form" method="post">
            <label for="name" className="field-label">Name</label>
            <input type="text" className="name-text-field w-input" maxLength="256" name="name" id="name-2" placeholder="Name" />
            <label for="email" className="field-label">Email Address</label>
            <input type="email" className="name-text-field w-input" maxLength="256" name="email" id="email-2" placeholder="Email Address" />
            <input id="contact-form-btnSubmit" type="submit" value="get started" data-wait="Please wait..." className="button-form w-button" />
        </form>
        <div className="w-form-done">
          <div>Thank you! We will contact you soon!</div>
        </div>
        <div className="w-form-fail">
          <div>Oops! Something went wrong while submitting the contact information, please try again.</div>
        </div>
      </div>
    </div>
  )
}