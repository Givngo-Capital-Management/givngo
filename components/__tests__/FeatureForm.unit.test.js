import React from 'react';
import { shallow } from 'enzyme';
import FeatureForm from '../Features/FeatureForm';

describe('FeatureForm', () => {
  test('renders', () => {
    const wrapper = shallow(<FeatureForm />);
    expect(wrapper).toMatchSnapshot();
  });
});
