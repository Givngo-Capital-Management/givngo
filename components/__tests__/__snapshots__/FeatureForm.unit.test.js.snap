// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeatureForm renders 1`] = `
<div>
  <div>
    <h2>
      Increase your
      <br />
      charity donations
    </h2>
    <div>
      Leave your contact information below, and we'll be in touch soon to help you get started
    </div>
    <form
      id="contact-form"
      method="post"
      name="email-form"
    >
      <label
        htmlFor="name-2"
      >
        Name
        <input
          className=" "
          id="name-2"
          maxLength="256"
          name="name"
          placeholder="Name"
          type="text"
        />
      </label>
      <label
        htmlFor="email-2"
      >
        Email Address
        <input
          className=" "
          id="email-2"
          maxLength="256"
          name="email"
          placeholder="Email Address"
          type="email"
        />
      </label>
      <input
        className=" "
        data-wait="Please wait..."
        id="contact-form-btnSubmit"
        type="submit"
        value="get started"
      />
    </form>
    <div>
      <div>
        Thank you! We will contact you soon!
      </div>
    </div>
    <div>
      <div>
        Oops! Something went wrong while submitting the contact information, please try again.
      </div>
    </div>
  </div>
</div>
`;
